Formula/little-cms.rb

```diff
@@ -24,6 +20,8 @@ class LittleCms < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c687f10094faebcec6573681241dc7f45e3f19ffcaab3ef2a56c987d13ba18a3"
   end
 
+  deprecate! date: "2021-09-28", because: :unmaintained
```

Comment:
#86070 added livecheck skip message, which is pretty much a deprecation message, so set to that date.

Reply:
Let's set it to today -- users won't have seen the deprecation notice from a skipped livecheck.